lib/binary-buffer.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+module.exports = () => {
+    /**
+     * Converts a string into a buffer. Does not change the input if it is
+     * already a buffer.
+     *
+     * @param {(Buffer|string)} stringOrBuffer
+     * @return {(Buffer|string)}
+     */
+    function toBuffer(stringOrBuffer) {
+        if (!Buffer.isBuffer(stringOrBuffer)) {
+            stringOrBuffer = new Buffer(stringOrBuffer.toString(), "binary");
```

Comment:
For a tiny speed boost (considering how often this is used, it might be good), I suggest changing `stringOrBuffer =` into `return`.